docs/docs/technology/architecture.md

```diff
@@ -0,0 +1,32 @@
+# Architecture
+
+Composable Finance is dedicated to **enhancing the cross-blockchain infrastructure in decentralized finance via trust-minimized bridging and order flow optimization**.
+
+Composable’s architecture is thoughtfully constructed to actualize this mission. This includes three different blockchains serving their own unique purposes. The architecture also includes trust-minimized bridging via the IBC protocol, which not only connects Composable’s blockchains, but connects other ecosystems as well. The Composable Technical stack also includes the Composable Virtual Machine (CVM) and MANTIS, which both serve roles in transaction execution.
+
+These different components collectively facilitate a complete, end-to-end ecosystem for cross-chain DeFi. In brief, the core components of the technical architecture are as follows:
+
+## [The CVM](cvm.md)
+In the blockchain industry, virtual machines play a key role in executing smart contracts, processing transactions, and ensuring the overall integrity of the chain. Existing virtual machines are designed for specific blockchains, such as the Ethereum Virtual Machine (EVM) being designed for the Ethereum chain.
+ 
+The Composable Virtual Machine (CVM) plays a different role in that it was constructed as a solution to carry out the critical role of a standard VM, but optimized for cross-chain operations. The CVM is an orchestration language and execution runtime for cross-chain program execution and intent settlement over IBC. With the CVM, developers are no longer restricted to one blockchain ecosystem and instead are able to execute cross-chain operations in one user-signed transaction.
```

Comment:
I doubt it will be standard VM. I mean it will be limited. We CVM program to express Intent it will not be standard VM. Will be very limited. Standard VMs are flexible to do anything. But that makes them non analyzable by algorithms to understand intent.

Reply:
edited